packages/react-jsx-highcharts/src/components/Axis/Axis.js

```diff
@@ -82,8 +60,29 @@ class Axis extends Component {
     }
   }
 
+  createAxis () {
```

Comment:
Think this method should be bound to `this`

```suggestion
  createAxis = () => {
```

Reply:
Why is that? It is not passed as parameter to anywhere.